packages/cli/src/config/auto-init/ApimlAutoInitHandler.ts

```diff
@@ -69,6 +69,18 @@ export default class ApimlAutoInitHandler extends BaseAutoInitHandler {
      * @throws {ImperativeError}
      */
     protected async doAutoInit(session: AbstractSession, params: IHandlerParameters): Promise<IConfig> {
+        // Login with token authentication first, so we can support certificates
+        if ((session.ISession.user && session.ISession.password) || (session.ISession.cert && session.ISession.certKey)) {
+            // If it is basic authentication, we need to set the auth type.
+            if (session.ISession.password) { session.ISession.type = SessConstants.AUTH_TYPE_TOKEN; }
+            session.ISession.tokenType = SessConstants.TOKEN_TYPE_APIML;
+            session.ISession.storeCookie = true;
+            session.ISession.tokenValue = await Login.apimlLogin(session);
+            session.ISession.storeCookie = false;
+            session.ISession.type = SessConstants.AUTH_TYPE_TOKEN;
+            session.ISession.user = session.ISession.password = session.ISession.cert = session.ISession.certKey = undefined;
+        }
```

Comment:
That will overwrite a token. It was not meant as an intentional change, and will provide a check for that.

Reply:
I took a look, and because of our prompting logic, only one set of these gets passed in (user/password, or tokenType/tokenValue, or cert/certKey, in that order). Also, because the base profile already exists, it will not overwrite the token in the base profile.